src/components/MessageList/__tests__/MessageList.test.js

```diff
@@ -13,18 +13,17 @@ import {
   generateUser,
   dispatchMessageNewEvent,
   getTestClientWithUser,
-} from 'mock-builders';
+} from '../../../mock-builders';
```

Comment:
`mock-builders` looks better, no?

Reply:
we have module resolver for this in jest.config.js

Reply:
Same for `ChannelList.test.js`

Reply:
ESLint was complaining about it and I overlooked our jest configuration. I'll adjust ESLint's configuration to fix it properly. Thanks! :D

Reply:
Done here: https://github.com/GetStream/stream-chat-react/pull/288/commits/80a0d733ad3b0c3112a851276ed2b7c15f83ff97